auslib/web/admin/views/problem.py

```diff
@@ -0,0 +1,44 @@
+import json
+from flask import Response
+
+
+def problem(status, title, detail, type=None, instance=None, headers=None, ext=None):
```

Comment:
It's unfortunate that you need to implement this yourself. What exactly went wrong with using Connexion's problem?